hal/src/emmc.c

```diff
@@ -1,50 +1,225 @@
 #include <hal/emmc.h>
-#include <drivers/sdmmc/fsl_mmc.h>
-#include <drivers/sdmmc/fsl_sdmmc_host.h>
+#include <boot/board.h>
+#include "MIMXRT1051.h"
 #include <errno.h>
 #include <stdbool.h>
 
+#include <drivers/fsl_iomuxc.h>
+#include <hal/delay.h>
+
+//#define DEBUG_USDHC
+#ifdef DEBUG_USDHC
+#include <drivers/fsl_src.h>
+
+#include "log.h"
+#include "eink.h"
+
+static void reportUSDHCRegisterSerial(void);
+static void reportUSDHCRegistersEink(status_t error);
+#endif
```

Comment:
I wonder if it is necessary?

Reply:
I used this code for debugging issues with eMMC and USDHC drivers. I decided to leave this code for the future. Who knows maybe some new issues will be discovered ;)